src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceRepository.kt

```diff
@@ -154,6 +155,18 @@ interface LicenceRepository :
   )
   fun getDraftLicencesPassedReleaseDate(): List<Licence>
 
+  @Query(
+    """
+    SELECT *
+        FROM licence
+        WHERE kind = 'HDC'
+        AND status_code IN ('IN_PROGRESS', 'SUBMITTED', 'APPROVED')
+        AND conditional_release_date - (INTERVAL '9' DAY) <= CURRENT_DATE;
+    """,
+    nativeQuery = true,
```

Comment:
Can I ask what the deal with `nativeQuery` is please? I haven't seen that in any of the others.

Reply:
So to answer the above comment and this, `nativeQuery` is the ability to use raw PSQL as your query as opposed to JPQL which allows you to query using your entity model. The reason for using a raw query here is JPQL doesn't support operations on dates so we can't do CRD-9 hence we have to use raw PSQL to do that. I have done a similar thing when getting the timed out licences for the time out licences job (see `getAllLicencesToTimeOut`)